signer/src/transaction_signer.rs

```diff
@@ -240,26 +240,21 @@ where
             ) => {
                 self.handle_stacks_transaction_sign_request(
                     request,
-                    &msg.bitcoin_chain_tip,
+                    &chain_tip,
                     &msg.signer_public_key,
                 )
                 .await?;
             }
 
             (message::Payload::WstsMessage(wsts_msg), _, _) => {
-                self.handle_wsts_message(
-                    wsts_msg,
-                    &msg.bitcoin_chain_tip,
-                    msg.signer_public_key,
-                    &chain_tip_report,
-                )
-                .await?;
+                self.handle_wsts_message(wsts_msg, msg.signer_public_key, &chain_tip_report)
```

Comment:
Previously here we were using the chain tip from the msg, now we are using the local chain tip. This kinda sounds better, but I'm not sure if it's messing things up (eg, when loading the `StateMachine`).
Kinda also applies to `BitcoinPreSignRequest` below, while the `StacksTransactionSignRequest` should be fine since we check that the message bitcoin ti[ is canonical.

Reply:
Yeah it makes a difference for sure. We probably need to make some other changes around there too, so I'll take a look.

But in general, I'm okay with us ignoring the old coordinator it the cases where we weren't.